docs/using-jda/using-new-features.md

```diff
@@ -0,0 +1,71 @@
+# Using New Features
+
+When contributors are making new features, like introducing modals, 
+you might be interested in trying out or testing these features before they get released.
+
+## Using [JitPack](https://jitpack.io)
+
+Using JitPack, you can get a build of any fork of JDA, on any branch, on any commit.
+It is still highly recommended that you use a build tool such as [Gradle](https://gradle.org) or [Maven](https://maven.apache.org)
+
+### Getting the Necessary Info
+
+Let's take the [PR for the Discord modals](https://github.com/DV8FromTheWorld/JDA/pull/2024) as an example
+
+![img.png](../assets/images/github_pr.png){width="720"}
+
+Here the important part is `from Xirado:feature/modals`, you need to extract the PR author, which here is `xirado`
+
+### Getting the Latest Version
+
+You then want to go in the `Commits` tab
+
+![img.png](../assets/images/github_pr_commits_tab.png){width="720"}
+
+You can now scroll down to the bottom, as to find the latest **commit hash**, it would look like this:
+
+![img.png](../assets/images/github_pr_latest_commit.png){width="720"}
+
+Here you can see the 7 first chars of the latest commit are `36eea2a`
+
+### Putting It Together
+
+So far we have:
+
+* PR author name: `xirado`
+* Version (7 char commit hash): `36eea2a`
+
+You can now add the dependency to your favorite build tool, but before that you will have to remove your current JDA dependency
+
+=== "Maven"
```

Comment:
Elsewhere in the wiki, Gradle has come before Maven.  This is probably something we should standardise.